api/AudioContext.json

```diff
@@ -150,6 +153,13 @@
             "ie": {
               "version_added": false
             },
+            "oculus": {
+              "notes": [
+                "Prior to Chrome 66, each tab is limited to 6 audio contexts in Chrome; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
```

Comment:
The note about Prior to Chrome 66 isn't relevant to Oculus browser.

Reply:
Fixed in b09cdc0a0bdda563df0249f86e2ba19eaf73065d.

In general Chrome notes often are relevant because Oculus Browser is a Chromium derivative and I have been translating Chrome versions to Oculus Browser ones. But in this case, M66 is ancient and the note is not relevant. Good catch! Thanks.